website/docs/docs/core/connect-data-platform/spark-setup.md

```diff
@@ -229,6 +233,10 @@ connect_retries: 3
 
 </VersionBlock>
 
+### Server side configuration
+
+Spark can be customized using [Application Properties](https://spark.apache.org/docs/latest/configuration.html). Using these properties the execution can be customized, for example, to allocate more memory to the driver process. Also, the Spark SQL runtime can be set through these properties. For example, this allows the user to [set a Spark catalogs](https://spark.apache.org/docs/latest/configuration.html#spark-sql).
+
```

Comment:
@Fokko is this fair?

```suggestion
<VersionBlock firstVersion="1.7">
### Server side configuration

Spark can be customized using [Application Properties](https://spark.apache.org/docs/latest/configuration.html). Using these properties the execution can be customized, for example, to allocate more memory to the driver process. Also, the Spark SQL runtime can be set through these properties. For example, this allows the user to [set a Spark catalogs](https://spark.apache.org/docs/latest/configuration.html#spark-sql).
</VersionBlock>
```

Reply:
@dataders Hope you had a great PTO.

If I'm not mistaken, it will be part of 1.6:

- https://github.com/dbt-labs/dbt-spark/pull/825
- https://github.com/dbt-labs/dbt-spark/pull/823
- https://github.com/dbt-labs/dbt-spark/pull/577

Reply:
that was our goal! but i don't think it happened. we're shooting for a `1.7b1` that includes these changes ASAP
https://github.com/dbt-labs/dbt-spark/commits/1.6.latest/

Reply:
Ah, I missed that cut. LGTM!